source/blog/2016-05-19-introducing-subteams.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Introducing Subteams
+author: Tom Dale
+tags: Recent Posts
+---
+
+I'm very happy to share with you our plan to scale Ember's day-to-day
+decision-making and coordination–something we're calling Subteams.
+
+Ember.js was once just a view layer for rendering templates in the
+browser, but has grown to become a complete SDK for the web. With one
+`npm install`, you get everything you need to create a modern web
+application.
+
+Making things simple for developers often means a lot of coordination
+and decision-making behind the scenes. As the number of pieces making up
+Ember has grown—encompassing not only Ember.js but Ember CLI, Ember
+Data, the Ember Inspector, documentation, and more—we want to ensure
+that the core team does not become a bottleneck for progress.
+
+At the same time, we believe that having a strong vision that everyone
+can rally behind is critical to building software that feels cohesive.
+So how do we ensure that everyone is pulling their cart in the same
+direction, without losing momentum?
+
+Today, we're taking a page from [Rust's
+playbook](https://internals.rust-lang.org/t/announcing-the-subteams/2042)
+and announcing Subteams. While we've often had informal teams tackle
+specific tasks, this change formalizes that process and officially
+recognizes the hard work of these contributors.
+
+## Core Team
+
+The core team serves as the leadership for the Ember project as a whole.
+Its responsibilities include:
+
+* **Setting the overall direction and vision for the project.** That means
+  setting the core values that are used when making decisions about
+  technical tradeoffs. It also means leading the discussion, and writing RFCs
+  for, major initiatives in the project.
+* **Setting priorities and the release schedule.** Design bandwidth is limited,
+  and it's dangerous to try to grow the framework too quickly; the core
+  team makes some difficult decisions about which areas to prioritize for
+  new design, based on the core values and target use cases.
+  The regular release cycle, being one of the project's core values,
+  applies to subteams as well.
+* **Focusing on broad, cross-cutting concerns.** The core team is specifically
+  designed to take a global view of the project, to make sure the pieces
+  are fitting together in a coherent way.
+* **Spinning up or shuting down subteams.** Over time, we may want to expand the
+  set of subteams, and it may make sense to have temporary "strike
+  teams" that focus on a particular, limited task.
+* **Going/no-going features**. While the subteams make decisions on RFCs,
+  the core team is responsible for pulling the trigger that moves a
+  feature from canary to beta. This provides an extra check that
+  features have adequately addressed cross-cutting concerns and that the
+  implementation quality is high enough
+
+Each subteam is lead by a core team liaison. This core team member
+is responsible for keeping the core team apprised of progress on
```

Comment:
Yep, I knew it was a word, just didn't know if that's the one you had intended ;-)